docs/configuration/jobs.rst

```diff
@@ -173,6 +173,74 @@ Note that you will have to trigger replication manually using the ``zrepl signal
        type: manual
      ...
 
+.. _job-snapshotting-hooks:
+
+Pre- and post-snapshot Hooks
+----------------------------
+
+Jobs with `periodic snapshots <job-snapshotting-spec_>`_ can run an executable before and/or after taking the snapshot,
+configured in an optional ``hooks`` section of the ``snapshotting`` configuration:
```

Comment:
Incorrect. `Periodic snapshotting can call optional pre- and post-hooks` (manual can't)

Reply:
But `manual` != `periodic`, or do I misunderstand some terminology? I meant to refer only to jobs with `snapshotting.type == "periodic"`.